config/config.go

```diff
@@ -30,46 +31,44 @@ const (
 
 // Config contains all the configuration needed.
 type Config struct {
-	Name string
-	Path string
+	Name string `validate:"required"`
+	Path string `validate:"required"`
 
 	Server struct {
-		Address string
+		Address string `validate:"required"`
 	}
 
 	Log struct {
-		Format      string
+		Format      string `validate:"required"`
```

Comment:
We could add `oneof=text json` > https://godoc.org/gopkg.in/go-playground/validator.v9#hdr-One_Of

With that, we can remove l149 and we could do the same for the logLevel